charts/consul/test/unit/helpers.bats

```diff
@@ -348,7 +348,7 @@ load _helpers
   [[ "$output" =~ "When the value global.experiments.resourceAPIs is set, global.peering.enabled is currently unsupported." ]]
 }
 
-@test "connectInject/Deployment: fails if resource-apis is set and admin partitions are enabled" {
+@test "connectInject/Deployment: fails if resource-apis is set, v2tenancy is unset, and admin partitions are enabled" {
```

Comment:
**suggestion:** Looks like extra stuff picked up.  `git checkout 'release/1.3.x' helpers.bats` will allow you to reset the file to the branch it is from.

Reply:
Corrected as recommended by reverting back to `release/1.3.x` branch version of affected files.

```shell
$ git checkout 'release/1.3.x' -- charts/consul/templates/server-disruptionbudget.yaml charts/consul/test/unit/server-disruptionbudget.bats charts/consul/template/_helpers.tpl
```
Applied datadog-integration changes back into `_helpers.tpl`

Re-ran entirety of bats tests using `Makefile` - `make bats-tests` (all passed)